webpack.config.js

```diff
@@ -1,89 +1,55 @@
-const { DefinePlugin } = require('webpack');
 const path = require('path');
+const { DefinePlugin } = require('webpack');
 const nodeExternals = require('webpack-node-externals');
 
-const assetConfig = {
-    devtool: 'source-map',
-    mode: 'production',
-    target: 'web',
-
-    entry: {
-        'checkout-sdk': './src/index.ts',
-        'checkout-button': './src/bundles/checkout-button.ts',
-        'embedded-checkout': './src/bundles/embedded-checkout.ts',
-        'internal-mappers': './src/bundles/internal-mappers.ts',
-    },
-
-    output: {
-        filename: '[name].js',
-        library: 'checkoutKit',
-        libraryTarget: 'commonjs2',
-        path: path.resolve(__dirname, 'dist'),
-    },
-
-    resolve: {
-        extensions: ['.ts', '.js'],
-    },
+const { babelLoaderRule, baseConfig, libraryEntries, libraryName } = require('./webpack-common.config');
 
-    module: {
-        rules: [
-            {
-                parser: {
-                    amd: false,
-                },
-            },
-            {
-                test: /\.(js|ts)$/,
-                include: path.resolve(__dirname, 'src'),
-                loader: 'ts-loader',
-            },
-        ],
-    },
-};
+const outputPath = path.join(__dirname, 'dist');
 
-module.exports = [
-    Object.assign({}, assetConfig, {
-        name: 'umd',
-        output: Object.assign({}, assetConfig.output, {
-            libraryTarget: 'umd',
+function getUmdConfig() {
+    return {
+        ...baseConfig,
+        entry: libraryEntries,
+        output: {
             filename: '[name].umd.js',
-        }),
-        module: Object.assign({}, assetConfig.module, {
+            library: libraryName,
+            libraryTarget: 'umd',
+            path: outputPath,
+        },
+        module: {
             rules: [
-                {
-                    test: /\.(js|ts)$/,
-                    include: path.resolve(__dirname, 'src'),
-                    loader: 'babel-loader',
-                    options: {
-                        presets: [
-                            [
-                                '@babel/preset-env',
-                                {
-                                    corejs: 3,
-                                    targets: [
-                                        'defaults',
-                                        'ie 11',
-                                    ],
-                                    useBuiltIns: 'usage',
-                                },
-                            ]
-                        ],
-                    },
-                },
-                ...assetConfig.module.rules,
+                babelLoaderRule,
+                ...baseConfig.module.rules,
             ],
-        }),
-    }),
+        },
+    };
+}
 
-    Object.assign({}, assetConfig, {
-        name: 'cjs',
```

Comment:
We need to keep the names for the task as it is being use to run them in isolation: 

https://github.com/bigcommerce/checkout-sdk-js/blob/master/package.json#L27-L28